File: lectures/002-hands-on/odd-even.py

# PYTHON PROGRAM

for j in range(0, 5) :
	even_counter = 0
	odd_counter = 0

	for i in range(0, 50000001):
		if i % 2 == 0 :
			even_counter += 1
		else :
			odd_counter += 1

	print ("even_counter "+ str(even_counter))
	print ("odd_counter "+ str(odd_counter))

File: lectures/002-hands-on/odd-even.c

#include <stdio.h> // C PROGRAM

int main() {
	for (int j=0; j<5; j++)
	{
		int even_counter = 0, odd_counter = 0, i = 0;
		for (i=0; i<50000001; i++) {
			if (i % 2 == 0 )
				even_counter += 1;
			else
				odd_counter += 1;
		}

		printf("even_counter %d\n", even_counter);
		printf("odd_counter %d\n", odd_counter);
	}
}
